src/common/regions/types.ts

```diff
@@ -110,7 +110,7 @@ export class MetroArea extends Region {
   }
 
   get fullName() {
-    return `${this.name}, ${this.stateCodes}`;
+    return `${this.name}`;
```

Comment:
I think the designs call for "New York-Newark-Jersey City, NY-NJ-PA" in the location title, we might need to keep this